SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+We aim to keep TimescaleDB safe for everyone. 
+Publicly disclosing security bugs in a public forum puts everyone in the Timescale community at risk.
+
+## Supported Versions
```

Comment:
@jfjoly do you have any other ideas?

Reply:
I again would cut this.  This "support policy" referenced is about paid commercial support, and the "supported version" you reference here refers to commercial support guarantees.  This is about contractual obligations, and have nothing to do with our open-source/community promises.  My feeling is that it will just raise fear/uncertaintly/doubt, for little benefit.d

Reply:
I checked the default policy template that Github provides (screenshot below). 
The goal of this section is to indicate which versions receive security fixes.

We could reword it with examples and be explicit that supported means providing security fixes: 
> We provide security fixes for the latest version of TimescaleDB. Minor versions (2.1 , 2.2...) don't get security fixes once a newer minor version is released. 
> For example, 2.6.0 is the most recent versions as of March 2022. New vulnerabilities will only be fixed against 2.6, there will be no backport to 2.5 or any older version.  

  
<img width="890" alt="Screen Shot 2022-03-23 at 18 32 26" src="https://user-images.githubusercontent.com/7732950/160561623-2dbaf1ec-1dc5-4263-9439-dc95c7121972.png">

Reply:
There was a previous discussion and this is why it was removed.